### PR TITLE
Han/contact

### PR DESCRIPTION
**Pull request by Hanchooi**
Basic Contact Form

### CODE REVIEW

**Review by noobling — APPROVED** [review]

**Review by Drewbi — CHANGES_REQUESTED** [review]
So I'm not sure why the website has that dark theme form but we definitely shouldn't copy that style as it doesn't make sense for the rest of the website. We also probably want to include the bio that's on the right of the contact page, unless you and Jeremiah decided not to include that intentionally.

**Comment by Hanchooi** [comment]
About " the bio that's on the right of the contact page" :
In the issue that Daphne raised up, she mentioned that we should remove them.

About "Dark Theme":
Good idea. Roger that.

File: client/components/contact/ContactForm.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<template>
+  <v-sheet tile elevation="0" color="info">
```
Comment (Drewbi) [inline client/components/contact/ContactForm.vue:2]: I appreciate the colour choice but in the interest of keeping this consistent with the rest of the site we should just keep this as a white background.

File: client/components/contact/ContactForm.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <v-form ref="form" v-model="valid" lazy-validation>
+        <v-text-field
+          v-model="firstname"
+          clearable
+          outlined
+          solo
+          :rules="nameRules"
+          required
+        >
+          <template #label>
```
Comment (Drewbi) [inline client/components/contact/ContactForm.vue:14]: I would use a label prop and change the styling using css instead of passing in a custom template.

**Comment by Hanchooi** [comment]
Changed to Light Themed, Added Title. Ready to merge.

File: client/components/contact/ContactHeader.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      background-image: url(https://bfarescue.com.au/wp-content/uploads/2015/08/rescue-yellow-dog.jpg);
+      background-size: cover;
+      height: 18.125rem;
+    "
+  >
+    <v-container>
+      <h1>
+        <p class="text-center">Got a question for us?</p>
+      </h1>
+      <h3>
```
Comment (Drewbi) [inline client/components/contact/ContactHeader.vue:14]: You shouldn't be wrapping you p tags in h3 tags and then style them as h6 tags. Use 1 of these methods, not all of them.

File: client/components/contact/ContactForm.vue [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          solo
+          :rules="mobileRules"
+          required
+        >
+          <template #label>
+            <label class="font-weight-bold">Mobile Number</label>
+          </template>
+        </v-text-field>
+        <v-textarea
+          v-model="message"
```
Comment (Drewbi) [inline client/components/contact/ContactForm.vue:52]: These fields haven't been added to the data yet.

**Review by Drewbi — CHANGES_REQUESTED** [review]

File: client/components/contact/ContactForm.vue (hunk 1 above)
Comment (Hanchooi) [reply client/components/contact/ContactForm.vue:2]: ok

File: client/components/contact/ContactHeader.vue (hunk 3 above)
Comment (Hanchooi) [reply client/components/contact/ContactHeader.vue:14]: uh oops. I don't know why I did this either. haha

File: client/components/contact/ContactForm.vue (hunk 4 above)
Comment (Hanchooi) [reply client/components/contact/ContactForm.vue:52]: Just added. But not sure if I did it correctly.

File: client/components/contact/ContactForm.vue (hunk 2 above)
Comment (Hanchooi) [reply client/components/contact/ContactForm.vue:14]: Ah, sorry. I don't know how to do it this way though. Couldn't find example using this method.

**Comment by Hanchooi** [comment]
Should be ready to merge.

**Review by definitelyDaphne — CHANGES_REQUESTED** [review]
Looking good overall
Noticed this little thing.. is there any reason why only the first name field has the clearable attribute?

Thanks

**Pull request merged**